### DISCUSSION TITLE
Neura chain onboarding proposal

### DISCUSSION THREAD

**Discussion by candostyavuz** [wormhole-foundation/wormhole#4081] [Gateway]
### Website

https://www.neuraprotocol.io/

### Github

Ankr Labs Github: https://github.com/Ankr-network/
Neura Chain Repo (Currently Private): https://github.com/Ankr-network/neura/

### Blog

NA

### Documentation

https://docs.neuraprotocol.io/protocol/technical-architecture

### Community stats

Twitter (https://x.com/Neura_io): 9942 followers
Telegram (https://t.me/neuraofficial): 1046 members
Discord (https://discord.com/invite/ATNu2VKgFZ): 1276 members

### Short Summary

Neura offers access to a network of decentralized GPU providers, allowing builders to launch and scale AI with flexible, affordable, on-demand compute. Neura is built using the Cosmos SDK which runs on top of the CometBFT to accomplish fast finality, high transaction throughput and short block times, but it is also fully compatible and interoperable with the Ethereum Virtual Machine (EVM).

### Why should Wormhole add this chain?

- Guardians are not required to run a full node for Neura - connectivity will be via a Gateway IBC connection.
- Use case: Primarily for bridging ANKR tokens (ERC-20, BEP-2, BEP-20) to Neura. The ANKR token serves as the primary currency within the Neura ecosystem.
- Ecosystem Details:
    * Ankr provides decentralized infrastructure across 50+ PoS networks
    * ANKR token market cap: $250,609,585 (as of August 16th, 2024)
    * Daily trading volume: $10,286,267 (as of August 16th, 2024)

### Technology and Features

- AI: A platform for advanced AI creation, training, and deployment
- Cloud: On-demand GPUs, Decentralized storage, Node infrastructure
- Cryptography: Bitcoin security, ZK proofs, Smart contract automation
- Web3: Crowdsourced funding, AI Compute-as-a-Currency, Advanced tokenomics

### Native token details

- Max Supply: 10,000,000,000 ANKR tokens
- Use Case: AI model usage, purchasing IMO tokens, staking, gas & fees

### Technical requirements for running a node

None - Neura will emit messages to Gateway via IBC. Guardians should not have to run anything additional (except for guardians who may run IBC relayers).

### IBC Relaying Strategy

We are requesting that one of the Wormhole Guardians handle the IBC relaying between Wormhole Gateway and Neura.

**Comment by PeaStew** [comment]
This is no longer relevant the conversation can be closed as the project has taken a different direction.
